capstone/capapi/views.py

```diff
@@ -28,35 +33,35 @@ class JurisdictionViewSet(viewsets.GenericViewSet, mixins.RetrieveModelMixin, mi
     renderer_classes = (renderers.BrowsableAPIRenderer, renderers.JSONRenderer)
 
 
-class VolumeViewSet(viewsets.GenericViewSet, mixins.RetrieveModelMixin, mixins.ListModelMixin,):
+class VolumeViewSet(BaseViewMixin, viewsets.GenericViewSet, mixins.RetrieveModelMixin, mixins.ListModelMixin):
```

Comment:
Do you need to have viewsets.GenericViewSet as an argument here when you've got the mixin?

Reply:
good catch!